src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -127,6 +147,14 @@ IFACEMETHODIMP_(void) FancyZones::Run() noexcept
         SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_UNAWARE);
         SetThreadDpiHostingBehavior(DPI_HOSTING_BEHAVIOR_MIXED);
     }}).wait();
+
+    if (RegOpenKeyEx(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\VirtualDesktops", 0, KEY_ALL_ACCESS, &m_virtualDesktopsRegKey) != ERROR_SUCCESS) {
+        return;
+    }
+    
+    m_fancyZonesDestroyedEvent.reset(CreateEvent(NULL, FALSE, FALSE, NULL));

```

Comment:
Be aware that there are some rare case where the Windows API will fail if passing `nullptr` instead of `NULL`. I don't recall which one I saw, but there are some. Don't ask me why they fail, I just know they do.

Reply:
yeah, I remember encountering them too :) they accept integers instead of pointers for some legacy reason.

Reply:
So should I change it to `nullptr` or not ? Reading documentation for these APIs, I've always encountered `NULL` and that's why I ussed it here.

Reply:
As a general rule of thumb, please use `NULL` only if `nullptr` doesn't compile.

Reply:
Sorry If I didn't explain myself: I meant do ahead, but be aware it may not work. If it doesn't revert it to NULL.